ramls/isbn.raml

```diff
@@ -0,0 +1,104 @@
+#%RAML 1.0
+title: Inventory API
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: "ISBN API"
+    content: <b>API for validation and conversion of ISBN-10 and ISBN-13 numbers </b>
+
+types:
+  errors: !include raml-util/schemas/errors.schema
+  isbn:
+    type: object
+    properties:
+      isbn: string
+    additionalProperties: false
+
+  isValid:
+    type: object
+    properties:
+      isValid: boolean
+    additionalProperties: false
+
+/isbn:
+  /convertTo13:
+    get:
+      description: Converts an ISBN code to an ISBN-13 code
+      queryParameters:
+        isbn:
+          displayName: Isbn
+          type: string
+          example: "091698477X"
+          required: true
+        hyphens:
+          displayName: Hyphens
+          type: boolean
+          default: false
+          required: false
+      responses:
+        200:
+          description: "The response body is a json object containing the converted isbn"
+          body:
+            application/json:
+              type: isbn
+              example: !include examples/isbn13.json
+        400:
```

Comment:
The standard API design uses [400 with plain text](https://github.com/folio-org/raml/blob/69f6074f5f48fd58e379e9ae76c892aa68c115ea/rtypes/item-collection.raml#L71), and [422 with JSON errors](https://github.com/folio-org/raml/blob/69f6074f5f48fd58e379e9ae76c892aa68c115ea/traits/validation.raml#L3).

What is different about the ISBN normalization API to mean that 400 with JSON is the chosen response?

Reply:
The main idea is that it is easier for the client to process the response in JSON format than to parse the text.